infra/nightly-resources/index.js

```diff
@@ -1,46 +1,194 @@
-// Top-level load function for the main index page.
-function load() {
-  // data should be in the format of Array<{runMethod: String, benchmark: String, total_dyn_inst: Int, hyperfine: Array<{results: **hyperfine `--json` results**}>}>
-  console.log(data);
-
-  let container = document.getElementById("profile");
-  let parsed = [];
-
-  // aggregate benchmark runs into a list by their "benchmark" key
-  let groupedBy = {};
-  data.forEach((obj) => {
-    if(!groupedBy[obj.benchmark]) {
-      groupedBy[obj.benchmark] = [];
+async function getPreviousRuns() {
+    const req = await fetch("https://nightly.cs.washington.edu/reports-json/eggcc/");
+    const files = await req.json();
+    
+    // map allLinks into an objects of the shape:
```

Comment:
what is `allLinks`? stale comment?

Reply:
Nice catch, I called it `allLinks` before I had a JSON API to work with, now it is `files`. I've updated the docs accordingly